lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -1549,6 +1549,21 @@ bool checkTypeForSPIRVExtendedInstLowering(IntrinsicInst *II, SPIRVModule *BM) {
     }
     break;
   }
+  case Intrinsic::abs: {
+        Type *Ty = II->getType();
+        int NumElems = 1;
+        if (auto *VecTy = dyn_cast<FixedVectorType>(Ty)) {
+            NumElems = VecTy->getNumElements();
+            Ty = VecTy->getElementType();
+    }
+        if ((!Ty->isIntegerTy()) ||
+        ((NumElems > 4) &&
+        (NumElems != 8) && (NumElems != 16))) {
+      `        BM->getErrorLog().checkError(false, SPIRVEC_InvalidFunctionCall,
+                                   II->getCalledOperand()->getName().str(), "",
+                                   __FILE__, __LINE__);
+    }
```

Comment:
This code can't be compiled by Travis CI https://travis-ci.org/github/KhronosGroup/SPIRV-LLVM-Translator/jobs/732684148#L559, because of some unexpected unicode symbols I guess.

Please follow the [LLVM coding standard](https://llvm.org/docs/CodingStandards.html#whitespace), in particular [this rule](https://llvm.org/docs/CodingStandards.html#golden-rule).